07-sampling.Rmd

```diff
@@ -73,128 +70,111 @@ percent_red_chr <- prop_red %>% percent(accuracy = 0.1)
 
 ## Sampling bowl activity {#sampling-activity}
 
-Let's start with a hands-on activity.
-
-
-### What proportion of this bowl's balls are red?
-
-Take a look at the bowl in Figure \@ref(fig:sampling-exercise-1). It has a certain number of red and a certain number of white balls all of equal size. `r if_else(is_latex_output(), '(Note that in this printed version of the book "red" corresponds to the darker-colored balls, and "white" corresponds to the lighter-colored balls. We kept the reference to "red" and "white" throughout this book since those are the actual colors of the balls as seen in the background of the image on our book\'s [cover](https://moderndive.com/images/logos/book_cover.png).)', '')` Furthermore, it appears the bowl has been mixed beforehand, as there does not seem to be any coherent pattern to the spatial distribution of the red and white balls.
-
-Let's now ask ourselves, what proportion of this bowl's balls are red?
+Take a look at the bowl in Figure \@ref(fig:sampling-exercise-1). It has a certain number of red balls and a certain number of white balls all of equal size. `r if_else(is_latex_output(), '(Note that in this printed version of the book "red" corresponds to the darker-colored balls, and "white" corresponds to the lighter-colored balls. We kept the reference to "red" and "white" throughout this book since those are the actual colors of the balls as seen in the background of the image on our book\'s [cover](https://moderndive.com/images/logos/book_cover.png).)', '')` The balls in the bowl have been mixed beforehand, as there does not seem to be any coherent pattern to the spatial distribution of the red and white balls. Our task is to determine, at least with some level of precision, the proportion of red balls in the bowl. 
```

Comment:
"spatial distribution" seems a bit too complex for the target audience here. Maybe something like "pattern to how the red and white balls fall in the hopper"?